shared/utility/math/angle.hpp

```diff
@@ -41,14 +41,14 @@ namespace utility::math::angle {
     template <typename T>
     inline T normalizeAngle(const T value) {
 
-        T angle = std::fmod(value, static_cast<T>(2.0 * M_PI));
+        T angle = std::fmod(value, static_cast<T>(M_2_PI));
 
         if (angle <= -M_PI) {
-            angle += M_PI * 2;
+            angle += M_2_PI;
         }
 
         if (angle > M_PI) {
-            angle -= 2 * M_PI;
```

Comment:
M_2_PI is `2/pi` not `2*pi` So these changes are no good!

Reply:
ahh yes. classic me